src/components/input/Demo.vue

```diff
@@ -175,6 +175,27 @@
                  type="number"
                  placeholder="Placeholder text."
                  label="Placeholder" />
+
+        <UIInput v-model="helpProp"
+                 :error="error"
+                 :small="small"
+                 :loading="loading"
+                 name="ui-text15"
+                 class="mt-10"
+                 help="Descriptive message..."
+                 label="Help (prop)" />
+
+        <UIInput v-model="helpSlot"
+                 :error="error"
+                 :small="small"
+                 :loading="loading"
+                 name="ui-text15"
+                 class="mt-10"
+                 label="Help (slot)">
+            <template #help>
+                <span class="animate-pulse text-sm mt-1 text-color-muted">Descriptive message...</span>
```

Comment:
no need for the pulse animation (bit distracting 😅), if you wanna sho it's custom just say `Slotted descriptive message...`